Implement validate-bundle script

galenatjpl opened the pull request:
This adds a new helper script, validate-parallel, which allows
for running validation in parallel.
The necessary, related documentation on how to use
validate-parallel is also included in this commit.

Resolves #128

galenhollins committed:
issue_128: Implement validate-parallel script

This adds a new helper script, validate-parallel, which allows
for running validation in parallel.
The necessary, related documentation on how to use
validate-parallel is also included in this commit.

jordanpadams commented on src/main/resources/bin/validate-parallel.sh:
```diff
@@ -0,0 +1,130 @@
+NUM_CORES=`getconf _NPROCESSORS_ONLN`
+if [[ "${NUM_GROUPS}" -eq "" ]]; then
+  echo "  Splitting up into groups, based on number of cores this machine has..."
+  NUM_GROUPS=`echo $NUM_CORES`
```
@galenatjpl after testing, does it make sense to use all cores? or was it more efficient to use ~1/2 of the cores available?

galenatjpl replied:
I updated the code to be half-cores by default, and committed this change (along with doc updates).

galenhollins committed:
issue_128:  update to half cores as default

Optimal performance found around half cores,
so making that the default.

jordanpadams committed:
Update assembly to include new validate-bundle script

jordanpadams committed:
Update script to include referential integrity

Adds validate execution for referential integrity (-R pds4.bundle).

Additionally, creates a consolidated summary report for the logs.
